antlr4/PostgreSQLLexer.g4

```diff
@@ -343,6 +347,7 @@ INTERVAL                             : I N T E R V A L;
 INTO                                 : I N T O;
 INVOKER                              : I N V O K E R;
 IS                                   : I S;
+ISNOTNULL                            : I S N O T N U L L;
```

Comment:
is this really a standalone symbol? Can you make sure you're checking against version 10.6 of the postgres documentation? I know that this is a little behind the latest, but I want to make sure this is done well before tweaking the grammar for later versions. I'm running the v10.6 of postgres on my local host, and this is what I see when I use `ISNOTNULL` vs `IS NOT NULL`:

```
postgres=# select 1 where 1 isnotnull;                                                                                                                          ERROR:  syntax error at or near "isnotnull"
LINE 1: select 1 where 1 isnotnull;
                         ^
postgres=# select 1 where 1 is not null;                                                                                                                         ?column? 
----------
        1
(1 row)
```

Reply:
Hmm. I has reverted ISNOTNULL. It was seems that with ISNOTNULL is better. 
Thank you!